src/main/java/core/basesyntax/service/OperationType.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public enum OperationType {
+    RETURN,
+    BUY,
+    SUPPLY;
+}
```

Comment:
It seems to me you can live without this enum, though it's not critical (in case you have another operation - you will need to change this enum in addition to other code. My understanding is you can use an enum for a fixed list of values which unlikely extend (like North, South, etc)